server.go

```diff
@@ -5096,3 +5103,34 @@ func (s *server) fetchClosedChannelSCIDs() map[lnwire.ShortChannelID]struct{} {
 
 	return closedSCIDs
 }
+
+// getStartingBeat returns the current beat. This is used during the startup to
+// initialize blockbeat consumers.
+func (s *server) getStartingBeat() (chainio.Beat, error) {
+	// beat is the current blockbeat.
+	var beat chainio.Beat
+
+	// We should get a notification with the current best block immediately
+	blockEpochs, err := s.cc.ChainNotifier.RegisterBlockEpochNtfn(nil)
```

Comment:
Could you add a comment why we add the nil blockepoch here ?

Reply:
done!